collector/pg_postmaster.go

```diff
@@ -44,7 +44,7 @@ var (
 		[]string{}, nil,
 	)
 
-	pgPostmasterQuery = "SELECT pg_postmaster_start_time from pg_postmaster_start_time();"
+	pgPostmasterQuery = "SELECT extract(epoch from pg_postmaster_start_time) from pg_postmaster_start_time();"
```

Comment:
I think this would be better to change our var to a sql.NullTime, but if we're pushing changes to this collector to another set of changes, this is fine for now. I opened #830 to track.